tests/Umbraco.Tests.Integration/Testing/UmbracoIntegrationTest.cs

```diff
@@ -188,6 +190,13 @@ protected virtual void ConfigureTestServices(IServiceCollection services)
     {
     }
 
+    /// <summary>
+    ///     Hook for registering umbraco test doubles.
+    /// </summary>
+    protected virtual void ConfigureUmbracoTestServices(IUmbracoBuilder builder)
+    {
+    }
```

Comment:
Why is this required? Seems to be the same as `CustomTestSetup` called below

Reply:
I didn't see that when I set up the test. I've switched to `CustomTestSetup` 😉